### PR TITLE
YAZ 5.34.1 breaks search with OCLC's "Sisis InfoGuide" origin

### PR DESCRIPTION
**Pull request by jakub-id**

### CODE REVIEW

File: IDMETA
Comment (adamdickmeiss) [inline IDMETA:1]: Hate pre releases. Up to you

File: src/cqltransform.c [hunk 1]
```diff
@@ -343,7 +343,7 @@ const char *cql_lookup_reverse(cql_transform_t ct,
                     Z_AttributeElement a_ae = *attributes->attributes[j];
                     if (!compare_attr(e_ae, &a_ae))
                         break;
-                    if (a_ae.attributeSet && e_ae->attributeSet &&
+                    if (a_ae.attributeSet &&
```
Comment (adamdickmeiss) [inline src/cqltransform.c:346]: Fine.

**Review by adamdickmeiss — APPROVED** [review]

File: IDMETA
Comment (jakub-id) [reply IDMETA:1]: Thanks. I'll make proper release once confirmed.

**Comment by jakub-id** [comment]
This has now been confirmed by ZIB to fix the problem. I will merge and make a final 5.34.2 release seperately.

**Pull request merged**